### PR TITLE
Roberto

### PR DESCRIPTION
**Pull request by s100tist**
Se agrega la función de impresión de PDF (Va una imagen para otra tarjeta de trello porque se me pasó:()

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
     "framer-motion": "^10.16.4",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-router-dom": "^6.16.0"
+    "react-router-dom": "^6.16.0",
+    "react-to-pdf": "^1.0.1",
+    "react-to-print": "^2.14.13"
```
Comment (Emanuel-Palestino) [inline package.json]: Is this dependency also used?

**Review by Emanuel-Palestino — CHANGES_REQUESTED** [review]
- The ```ToolDescription.tsx``` file shouldn't have been modified.
- Why are there empty files?
- Name files and write code in English please.

File: package.json (hunk 1 above)
Comment (s100tist) [reply package.json]: It isn't used, it already got removed in my last push

File: package-lock.json
Comment (s100tist) [inline package-lock.json:1]: What's the purpose of this review?

**Review by Emanuel-Palestino — APPROVED** [review]

**Pull request merged**